lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
+      color: var(--nx-color-vulnerability-#{$severity}-text);
+      border: 1px solid var(--nx-color-vulnerability-#{$severity}-border);
+    }
+  }
+}
+
+.nx-small-vulnerability-counter {
+  $height: 25px;
+  position: relative;
+  border-radius: 2px;
+  font-size: var(--nx-font-size-s);
+  font-weight: 600;
+  height: $height;
+  line-height: $height;
+  padding-left: var(--nx-spacing-base);
+  padding-right: var(--nx-spacing-base);
+  text-align: center;
+
+  &__severity {
+    //invisible but still read by the screenreader
+    @include nx-text-helpers.visually-hidden;
+  }
+
+  @include counter-colors;
+
+  // Used to set the width of the counter based on its max text content
+  &__sizer {
+    display: block;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+
+  &--zero {
+    background-color: var(--nx-color-small-vulnerability-counter-zero-background);
+    color: var(--nx-color-small-vulnerability-counter-zero-text);
+    border: 1px solid var(--nx-color-small-vulnerability-counter-zero-border);
+  }
+}
+
+.nx-small-vulnerability-counter-container {
+  display: inline-flex;
+  gap: var(--nx-spacing-2x);
+  margin-left: var(--nx-spacing-2x);
+  margin-right: var(--nx-spacing-2x);
```

Comment:
I don't see a spec for this though I do see that it matches `NxSmallThreatCounter`.

Reply:
> I don't see a spec for this though I do see that it matches `NxSmallThreatCounter`.

Just confirmed: this should be 8px as is

Reply:
> Just confirmed: this should be 8px as is

That needs to be in the spec

Reply:
> > Just confirmed: this should be 8px as is
> 
> That needs to be in the spec

@Kiran-softwaretest

Reply:
The figma mockup has been modified with the specs